docs/source/release_notes/release-notes.rst

```diff
@@ -96,6 +96,11 @@ v0.53.0
          - IteratorUtils.forEach removed; it's not needed in a Java 8 codebase.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2207>`__)
 
+    *    - |improved|
+         - New efficient and OOM-proof implementation of sweep for Cassandra KVS.
```

Comment:
If we have figures on how many round-trips this is likely to save, it would be good to mention it here.